video/out/wayland_common.c

```diff
@@ -41,6 +42,25 @@
 // Generated from presentation-time.xml
 #include "video/out/wayland/presentation-time.h"
 
+#define OPT_BASE_STRUCT struct wayland_opts
+const struct m_sub_options wayland_conf = {
+    .opts = (const struct m_option[]) {
+        OPT_CHOICE("wayland-frame-timing", frame_timing, 0,
+                    ({"auto", -1}, {"presentation-time", 0},
+                    {"presentation-time-no-ust", 1},
+                    {"frame-callback", 2})),
```

Comment:
Is so much control really necessary? GLX/x11-EGL don't have options to disable use of vsync feedback either.